docs/cristin.ttl

```diff
@@ -906,10 +906,75 @@ It is a self reference.""" ;
         entityDescription.reference.doi
     """ .
 
+:antall_sider_totalt
+    a               :cristinfield ;
+    rdfs:comment    "The total number of pages. Possibly null." ;
+    rdfs:range      rdf:nil ; #unknown range
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:publiseringsar
+    a               :cristinfield ;
+    rdfs:comment    "The year of publication" ;
+    rdfs:range      xsd:positiveInteger ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:hefte
+    a               :cristinfield ;
+    rdfs:comment    "The booklet containing the publication." ;
+    rdfs:range      rdf:nil ; #unknown range
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:status_referee_ordning
+    a               :cristinfield ;
+    rdfs:comment    "Defines whether or not the Journal is peerreviewed (J)=yes, (N)=no and NULL." ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:supplement
+    a               :cristinfield ;
+    rdfs:comment    "A area for adding information about possible attachments to the article." ;
+    rdfs:range      rdf:nil ; #unknown range
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:utbredelsesomrade
+    a               :cristinfield ;
+    rdfs:comment    """
+        Information about where or to what extent the article has been published.
+        Examples: NORSK, NORDISK, INTERNASJ, NASJONALT, UKJENT""" ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:artikkelnummer
+    a               :cristinfield ;
+    rdfs:comment    "The article number." ;
+    rdfs:range      rdf:nil ; #unknown range
+    rdfs:domain     :TYPE_TIDSSKRIFTPUBLIKASJON_Class ;
+    :isignored   "true" .
+
+:arstall_trykket
+    a               :cristinfield ;
+    rdfs:comment    "The year the publication was printed." ;
+    rdfs:range      rdf:nil ; #unknown range
```

Comment:
Use SQL with regular expressions to find whether the range is number, number as string, or free text